server/model/board.go

```diff
@@ -359,5 +360,5 @@ type BoardMemberHistoryEntry struct {
 
 	// The insertion time
 	// required: true
-	InsertAt int64 `json:"insertAt"`
+	InsertAt time.Time `json:"insertAt"`
```

Comment:
Do we need this in the model?  Generally we're just using `insert_at` to get a consistent order from queries.  If we need it then store needs to make the conversion.

Reply:
This is for the membership history, so... the whole point is to expose the join/leave date here. We can expose it in other way, but we need to expose it (only for admins, of course, and I think is going to be behind the license).